astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +38,260 @@ The login flow will look like this after FusionAuth is introduced.
+</Aside>
 
-### Overview of the Application
+### Create A Basic Spring Boot Application
 
-We are going to be building a version of the ChangeBank app. This app will allow you to log in and see your "balance", as well as present a page that you can input a dollar amount and have that broken into change of dimes and pennies. You won't be keeping track of any account state beyond the user in FusionAuth, so the only actual functionality you will implement is the api call to make change.
+Next, you'll set up a basic Spring Boot project with a single app. While this guide builds a new Spring Boot project, you can use the same method to integrate your existing project with FusionAuth.
 
-You are going to build an application with three views:
-- `Home`: This is the welcome page than an unauthenticated user will see when first visiting your application.
-- `Account`: This is a protected view the user can only see after they have logged in.
-- `Make Change`: This view is also protected and demonstrates making an authenticated form post to the backend.
+#### Get The Template
 
-### Downloading the template from Spring Initializr
+The template is already downloaded in the example repository, but if you wish to download it yourself, go to the Initializr site at https://start.spring.io/ and download your own starter package.
 
-We have already done this for you in the example repository, but if you wish to do so yourself you can go to the Initializr site at https://start.spring.io/ and download your own starter package. We will rely on three dependencies for this project:
+You can unzip the package in any directory you would like to. This guide assumes you are in `stub-application` in the example repository, so you should substitute any references to that directory with the directory you unzipped the files in.
 
-- OAuth2 Client
-- Thymeleaf
-- Spring Web
+This example was made with Maven, Spring Boot version 2.7.12, and Java 17. If you use different options, the configuration and code may be different.
 
-This example was made with `Maven`, Spring Boot version `2.7.12`, and Java 17. If you choose different options the configuration and code may be different.
+#### Install Dependencies And Build The Application
 
-If you choose you can download and unzip the package in any directory you wish and proceed to follow along with the rest of this guide. The directions will assume you are in `stub-application` from the example repository. Please substitute that directory with wherever you unzipped the files from Spring.
+This project has three dependencies:
 
-### Install the dependencies
+- OAuth2 Client
+- Thymeleaf
+- Spring Web
 
-From the `stub-application` directory run the following to execute the Maven build tool to download dependencies and build your application.
+From the `stub-application` directory, run the following to execute the Maven build tool to download dependencies and build your application.
 
 ```shell
 ./mvnw package
```

Comment:
I am not familiar with Java and when I try to run this, I get the following error:

`./mvnw package
./mvnw: line 280: /Users/mark.robustelli/Projects/quickstart-springboot-web/fusionauth-quickstart-springboot-web/stub-application/.mvn/wrapper/maven-wrapper.properties: No such file or directory
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0Warning: Failed to open the file 
Warning: /Users/mark.robustelli/Projects/quickstart-springboot-web/fusionauth-q
Warning: uickstart-springboot-web/stub-application/.mvn/wrapper/maven-wrapper.j
Warning: ar: No such file or directory
 11 58727   11  6886    0     0  15063      0  0:00:03 --:--:--  0:00:03 15134
curl: (23) Failure writing output to destination
Error: Could not find or load main class org.apache.maven.wrapper.MavenWrapperMain
Caused by: java.lang.ClassNotFoundException: org.apache.maven.wrapper.MavenWrapperMain
`
Am I missing something or is there something I need to do to work around this?